### DISCUSSION TITLE
Problem through YOUR SECOND QUARKUS APPLICATION tutorial

### DISCUSSION THREAD

**Discussion by guipinheiro** [quarkusio/quarkus#41229] [Q&A]
Hello, everyone.

So, I was doing the `YOUR SECOND QUARKUS APPLICATION` tutorial when I reached section 3 and tried to install both panache and postgres extensions at the same time using mvn, however, I was not able to and received this error message running the command `./mvnw quarkus:add-extension -Dextensions='hibernate-orm-panache jdbc-postgresql'`:

```shell
[INFO] [ERROR] ❗  Nothing installed because keyword(s) 'hibernate-orm-panache jdbc-postgresql' were not matched in the catalog.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.412 s
[INFO] Finished at: 2024-06-15T17:31:36-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:3.11.2:add-extension (default-cli) on project getting-started-dev-services: Unable to update the pom.xml file: Unable to add extensions -> [Help 1]
[ERROR] 
```
My solution was installing one extension at a time.

`./mvnw quarkus:add-extension -Dextensions='hibernate-orm-panache'`
`./mvnw quarkus:add-extension -Dextensions='jdbc-postgresql'`
Is this method of installing multiple extension at once not working using maven?

**Comment by maxandersen** [comment]
That looks like a bug. Multiple should work.

**Comment by andleosab** [comment]
This worked:
./mvnw quarkus:add-extension -Dextensions='hibernate-orm-panache,jdbc-postgresql'

**Reply by cescoffier** [reply]
yes, the colon is missing. Now we could be a bit smarter and consider space as a separator.

**Comment by gsmet** [comment]
I added the comma in https://github.com/quarkusio/quarkus/pull/43743 .

Maybe we should support spaces too as previously mentioned but at least the doc will be right for now. Thanks for taking the time to report it!
